LICENSE.txt

```diff
@@ -202,16 +202,21 @@
    limitations under the License.
 
 --------------------------------------------------------------------------------
-vector/src/main/java/org/apache/arrow/vector/util/IntObjectHashMap.java
-vector/src/main/java/org/apache/arrow/vector/util/IntObjectMap.java
+This product includes code from Netty 4.1.117.Final:
```

Comment:
Is it required to include the specific version? It would be easily out of sync since the dependabot does not maintain this.

Reply:
Unfortunately (or not 😄 ), yes,  It's important to specify the version of the dependency as licenses sometimes change as product versions change.
Also the `NOTICE` can change from a version to another.
So, it's better to document to actual version bundled in our distributed jar.

So source distribution, we should document the version where the code has been copied from.

Reply:
Thanks for the explanation! If we don't have good automation tools to keep them in sync, at least we need to make sure they are accurate in the release process.